libs/components/src/lib/conditions-card/conditions-card.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { CardWrapper, NewButton, NewCard } from './styles';
+
+import 'antd/dist/antd.css';
+
+export function ConditionsCard(): ReactElement {
+  const navigate = useNavigate();
+  const [dis, setDisabled] = useState(false);
+  const card = React.useRef<HTMLDivElement>(null);
```

Comment:
I dont like refs, and I see only one case when we could have it. so my question is why do we need 2 refs in this component

Reply:
fixed